SpatialGDK/Extras/schema/rpc_payload.schema

```diff
@@ -11,4 +11,5 @@ type UnrealRPCPayload {
     uint32 rpc_index = 2;
     bytes rpc_payload = 3;
     option<TracePayload> rpc_trace = 4;
+    uint32 rpc_unique_id = 5;
```

Comment:
Is 32 bits enough for this and do we want to call it unique? I guess this is a temporary measure so it's probably not that important. Are collisions are double sending worse?

Reply:
I can increase it to 64 bits, if you want?